install/uninstall.go

```diff
@@ -115,6 +115,16 @@ func (k *K8sUninstaller) Uninstall(ctx context.Context) error {
 			time.Sleep(retryInterval)
 			goto retry
 		}
+
+		k.Log("⌛ Waiting for %s namespace to be terminated...", k.params.TestNamespace)
+	retryNamespace:
+		// Wait for the test namespace to be terminated. Subsequent connectivity checks would fail
+		// if the test namespace is in Terminating state.
+		_, err = k.client.GetNamespace(ctx, k.params.TestNamespace, metav1.GetOptions{})
```

Comment:
Hmm... Instead of the busy wait loop, I wonder if we can use the `wait` functionality exposed by kubectl - https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#wait.

>Alternatively, the command can wait for the given set of resources to be deleted by providing the "delete" keyword as the value to the --for flag.

Reply:
agreed it's better to use watch or something instead. do you want to tackle that in a separate PR? right now we do busy wait everywhere: https://github.com/cilium/cilium-cli/blob/master/connectivity/check/deployment.go would be better to get all fixed in a dedicated PR.

Reply:
Ok, I've filed - https://github.com/cilium/cilium-cli/issues/374.